cdap-app-fabric/src/main/java/co/cask/cdap/internal/provision/NativeProvisioner.java

```diff
@@ -78,7 +103,7 @@ public PollingStrategy getPollingStrategy(ProvisionerContext context, Cluster cl
   }
 
   @Override
-  public Set<String> getCapabilities() {
-    return Collections.singleton(Requirements.TEPHRA_TX);
+  public Capabilities getCapabilities() {
+    return new Capabilities(TRANSACTIONAL_DATASETS);
```

Comment:
nit: This Capabilities instance can actually be the constant instead of TRANSACTIONAL_DATASET.